apps/web/src/pages/TermsAndConditions/data.ts

```diff
@@ -0,0 +1,39 @@
+export const data = [
```

Comment:
please rename this to `TERMS_AND_CONDITIONS` and move it to the const file.

Reply:
done